browser/brave_browser_process.h

```diff
@@ -87,6 +88,8 @@ class BraveBrowserProcess {
 #endif
   virtual debounce::DebounceComponentInstaller*
   debounce_component_installer() = 0;
+  virtual brave::URLSanitizerComponentInstaller*
+  url_sanitizer_component_installer() = 0;
```

Comment:
Nit: Fixing this might not be possible other functions around are violating the style too, but these virtual functions should be using `PascalCase` if we were to follow chromium style here, since `unix_case` is used in chromium to indicate that a function is inlined when called, which won't be the case for these virtuals.

Reply:
done, thanks